css/css-values/calc-in-media-queries-with-mixed-units.html

```diff
@@ -7,17 +7,23 @@
     <link rel="help" href="https://www.w3.org/TR/css3-values/#calc-computed-value">
   </head>
   <body>
-    <iframe src="./support/mixed-units-01.html" title="px/em" frameborder="0" height="10" width="100"></iframe>
-    <iframe src="./support/mixed-units-02.html" title="vh/em" frameborder="0" height="10" width="100"></iframe>
-    <iframe src="./support/mixed-units-03.html" title="vw/em" frameborder="0" height="10" width="100"></iframe>
-    <iframe src="./support/mixed-units-04.html" title="vw/vh" frameborder="0" height="10" width="100"></iframe>
-    <iframe src="./support/mixed-units-05.html" title="vh/px" frameborder="0" height="10" width="100"></iframe>
-    <iframe src="./support/mixed-units-06.html" title="vw/px" frameborder="0" height="10" width="100"></iframe>
+    <iframe src="./support/mixed-units-01.html" title="px-em" frameborder="0" height="10" width="100"></iframe>
+    <iframe src="./support/mixed-units-02.html" title="vh+em" frameborder="0" height="10" width="100"></iframe>
+    <iframe src="./support/mixed-units-03.html" title="vw-em" frameborder="0" height="10" width="100"></iframe>
+    <iframe src="./support/mixed-units-04.html" title="vw+vh" frameborder="0" height="10" width="100"></iframe>
+    <iframe src="./support/mixed-units-05.html" title="vh+px" frameborder="0" height="10" width="100"></iframe>
+    <iframe src="./support/mixed-units-06.html" title="vw+px" frameborder="0" height="10" width="100"></iframe>
+    <iframe src="./support/mixed-units-07.html" title="px/em*em" frameborder="0" height="10" width="100"></iframe>
+    <iframe src="./support/mixed-units-08.html" title="vh*em" frameborder="0" height="10" width="100"></iframe>
+    <iframe src="./support/mixed-units-09.html" title="vh*vw/em*px/vh" frameborder="0" height="10" width="100"></iframe>
+    <iframe src="./support/mixed-units-10.html" title="vw/px*vh" frameborder="0" height="10" width="100"></iframe>
+    <iframe src="./support/mixed-units-11.html" title="vh*vw/em*px" frameborder="0" height="10" width="100"></iframe>
+    <iframe src="./support/mixed-units-12.html" title="vw*vh*px*em/px/px/px" frameborder="0" height="10" width="100"></iframe>
     <script>
       for (const frame of document.querySelectorAll("iframe")) {
         async_test((t) => {
           frame.addEventListener("load", t.step_func(() => {
-            const box = frame.contentWindow.document.querySelector(".box");
+            const body = frame.contentWindow.document.body;
             const actual = frame.contentWindow.getComputedStyle(box).getPropertyValue("background-color");
```

Comment:
```suggestion
            const actual = frame.contentWindow.getComputedStyle(body).getPropertyValue("background-color");
```

Reply:
Had no idea you could suggest changes like that. Cool!

Wish I could get some sort of linting working on this repo

Reply:
Resolved in [6fc42d2](https://github.com/web-platform-tests/wpt/pull/42517/commits/6fc42d2f58f2f1d373c017a310256b7c215f25af)

Reply:
There is `./wpt lint`, but it's extremely basic, I don't think it would have caught this.